src/main/java/com/stripe/model/Refund.java

```diff
@@ -24,9 +24,14 @@
   Long created;
   String currency;
   String description;
+  @Getter(AccessLevel.NONE) @Setter(AccessLevel.NONE)
+      ExpandableField<BalanceTransaction> failureBalanceTransaction;
+  String failureReason;
   @Getter(onMethod = @__({@Override})) Map<String, String> metadata;
   String reason;
   String receiptNumber;
+  @Getter(AccessLevel.NONE) @Setter(AccessLevel.NONE)
+      ExpandableField<Reversal> sourceTransferReversal;
```

Comment:
you're missing `transfer_reversal` https://stripe.com/docs/api/refunds/object#refund_object-transfer_reversal